pandas/tests/window/test_rolling.py

```diff
@@ -1707,7 +1710,7 @@ def test_rolling_quantile_interpolation_options(quantile, interpolation, data):
     if np.isnan(q1):
         assert np.isnan(q2)
     else:
-        assert q1 == q2
+        assert np.allclose([q1], [q2])
```

Comment:
Did this change on all platforms? Or just a 32 bit thing? If the latter thing would be better to use compat

Reply:
Just 32 bit. Isn't it a bad idea to compare float arrays with == though?

Reply:
These should still work with the normal equality. The precision of a 32 bit integer should still exceed what allclose does, so something is not right here. 32 bit xfail is fine for now

Reply:
Reverted.